first_hour_on_vwb/working_with_data.ipynb

```diff
@@ -0,0 +1,589 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Working with data\n",
+    "\n",
+    "<table align=\"left\">\n",
+    "\n",
+    "  <td>\n",
+    "    <a href=\"https://github.com/DataBiosphere/terra-axon-examples/blob/main/first_hour_on_vwb/working_with_data.ipynb\">\n",
+    "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\">\n",
+    "      View on GitHub\n",
+    "    </a>\n",
+    "  </td>\n",
+    "</table>\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "tvgnzT1CKxrO"
+   },
+   "source": [
+    "## Overview\n",
+    "\n",
+    "This notebook provides examples of working with referenced resource data. Build upon the best practices demonstrated in this notebook to visualize and analyze data in your own workspaces.\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "d975e698c9a4",
+    "tags": []
+   },
+   "source": [
+    "### Objective\n",
+    "\n",
+    "Use this notebook to perform common workspace resource operations including:\n",
+    "\n",
+    "- [Access data from referenced resources](#access-ref-data)\n",
+    "  - [Read data from BigQuery](#read-from-bq)\n",
+    "    - [Using pandas-gbq library](#using-pandas)\n",
+    "    - [Using google-cloud-bigquery](#using-gcb)\n",
```

Comment:
Double check these links, I think the read-from-bq and using-gcb links need to be fixed

Reply:
fixed!